packages/nodes-base/nodes/Twist/ThreadDescription.ts

```diff
@@ -0,0 +1,630 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const threadOperations = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		displayOptions: {
+			show: {
+				resource: [
+					'thread',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Add',
+				value: 'add',
+				description: 'Adds a new thread to a channel.',
```

Comment:
```suggestion
				description: 'Add a new thread to a channel.',
```

Reply:
Changed it to `Create` to be consistent with the other operations